todo_app/posts/api.py

```diff
@@ -0,0 +1,29 @@
+from posts.models import Post, Topic
+from rest_framework import viewsets, permissions
+from .serializers import PostSerializer, TopicSerializer, UserSerializer
+from django.contrib.auth.models import User
+# Lead Viewset
+
+
+class PostViewSet(viewsets.ModelViewSet):
+    # permission_classes = [
+    #     permissions.IsAuthenticated,
+    # ]
+    serializer_class = PostSerializer
+
+    def get_queryset(self):
+        return Post.objects.all()
+        # return self.request.user.posts.all()
```

Comment:
In this case, since there's not much computation, you can say:
```
serializer_class = PostSerializer
queryset = Post.objects.all()
```
and remove your `get_queryset` method.